server/src/database/schema.prisma

```diff
@@ -788,6 +788,7 @@ model Favorite {
   /// @TypeGraphQL.omit(input: true, output: false)
   workspaceMember   WorkspaceMember? @relation(fields: [workspaceMemberId], references: [id])
   workspaceMemberId String?
+  index             Int?             @default(0)
```

Comment:
let's call it "position". I know that we have been using "index" in the past for other objects but then the naming is painful to use in the frontend because it conflicts with the JS term array "index".